pootle/templates/staticpages/_body.html

```diff
@@ -9,7 +9,14 @@
   {% endblocktrans %}
   </div>
   {% endif %}
-  <h1 class="title">{{ page.title }}</h1>
+  <h1 class="title">
+    {{ page.title }}
+    {% if user.is_superuser %}
```

Comment:
Just noticed that perhaps the admin UI for editing announcements must be also adjusted so the displayed announcements are restricted to those having only the required permissions.